pkg/controller/studyjob/utils.go

```diff
@@ -39,6 +40,70 @@ func createWorkerJobObj(kind string) runtime.Object {
 	return nil
 }
 
+func validateWorkerResource(wkind string) error {
+	for _, crd := range invalidCRDResources {
+		if crd == wkind {
+			return fmt.Errorf("Cannot support %s; Please install the CRD and restart studyjob-controller", wkind)
+		}
+	}
+	return nil
+}
+
+func ignoreWatchError(err error, job string) bool {
```

Comment:
The name of the function is a little confusing. Maybe change it to "isFatalWatchError" and return the opposite boolean values?

Reply:
Thanks! Done!